### PR TITLE
Fix[Core/Instance] Culling_of_stratholme Chromie give wrong item repeatly

### PR DESCRIPTION
**Pull request by betayi**
Fix Chromie add wrong item 37889 to player cause of cast a wrong spell

## Changes Proposed:
-  as title
-  

## Issues Addressed:

- Closes 

## Tests Performed:

Tested

## How to Test the Changes:

1.make a chracters that has finish Quest 13149
2.Tele to Culling_of_Stratholme
3.speak to chromie
3.see if she give you item 37888 but not repeatly 37889

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by pklloveyou** [comment]
image
The wrong data is here

**Comment by betayi** [comment]
> image The wrong data is here

you are right,  and i also exclained this in the first line of this issue content

**Comment by betayi** [comment]
did you means correct the customized spell's effectitemtype field with 37888 to replace additem 37888 ?

**Comment by Kitzunu** [comment]
It's not a custom spell, it is just a serverside spell. Blizzard often added items with spells and not by just adding items.

So in this case, you fix the data in ApplySpellFix and you should not just additem

File: src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/culling_of_stratholme.cpp [hunk 1]
```diff
@@ -1444,7 +1444,7 @@ class npc_cos_chromie_start : public CreatureScript
 
                 if (!player->HasItemCount(ITEM_ARCANE_DISRUPTOR))
                 {
-                    me->CastSpell(player, SPELL_SUMMON_ARCANE_DISRUPTOR);
+                    player->AddItem(ITEM_ARCANE_DISRUPTOR, 1);
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/culling_of_stratholme.cpp:1447]:
```cpp
me->CastSpell(player, SPELL_SUMMON_ARCANE_DISRUPTOR);
```
is correct. Fix the spell

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/culling_of_stratholme.cpp (hunk 1 above)
Comment (betayi) [reply src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/culling_of_stratholme.cpp]: got it

**Review by Kitzunu — APPROVED** [review]

**Comment by claudiodfc** [comment]
Test result: ✔

- After quest complete and talking to the NPC the first gossip options, Chromie gives you the item 37888 

image

**Pull request merged**